core/modules/qana/PostPlugin.cc

```diff
@@ -75,8 +75,14 @@ class PostPlugin : public QueryPlugin {
 
     int _limit;
     std::shared_ptr<query::OrderByClause> _orderBy;
+
+private:
+	static LOG_LOGGER logger;
```

Comment:
I'm not sure that logger has to be class-static variable. It looks like your pattern in other files is to have per-file logger instead of per-class. In that case you can put logger outside of the class in an anonymous namespace.